package/MDAnalysis/coordinates/chain.py

```diff
@@ -341,21 +341,21 @@ def __init__(self, filenames, skip=1, dt=None, continuous=False,
             n_frames = 0
             for r1, r2 in zip(self.readers[:-1], self.readers[1:]):
                 r2[0], r1[0]
-                r1_start_time = r1.time
```

Comment:
@richardjgowers I'm no chainreader expert, is `ts.time` required or should you just be able to use `.time`

Reply:
No you're right, I'll fix this.